CCS/Hubunit_Front/src/init.c

```diff
@@ -63,8 +64,18 @@ static void create_tasks(void) {
 /*
  *  ======== main ========
  */
+
+UBaseType_t length;
+UBaseType_t size;
+
+
 int main(void)
 {
+    length = 0; //Replace with real size
```

Comment:
could turn length into a macro. Maybe create a header file for all the macros that will be used for this project.